dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -282,17 +282,19 @@ object LagomPlugin extends AutoPlugin {
     val lagomDevSettings = settingKey[Seq[(String, String)]]("Settings that should be passed to a Lagom app in dev mode")
     val lagomServicePort = taskKey[Int]("The port that the Lagom service should run on")
 
+    val lagomInfrastructureServices = taskKey[Seq[Task[Closeable]]]("The infrastructure services that should be run when runAll is invoked.")
```

Comment:
This is the new hook for starting "infrastructure" services. By default, it is configured to include Kafka, Cassandra and the service locator.